src/pages/index.tsx

```diff
@@ -29,7 +29,7 @@ const HomePage: NextPage<
 > = ({ appUrl }) => {
   return (
     <div className="h-screen w-screen bg-gradient-to-br from-pink to-orange p-2 lg:p-4">
-      <Seo />
+      <Seo description="Fortress provides composable financial products for passive DeFi investors" />
```

Comment:
Set this as the default description in Seo.tsx

Reply:
done